recipe/meta.yaml

```diff
@@ -1,34 +1,38 @@
-{% set version = "1.14.1" %}
+{% set version = "1.15.0" %}
 
+# Update the numpy value here to the most recent released version
+# when updating to a new scipy feature release
 {% if numpy is not defined %}
-{% set numpy = "2.0.0" %}
+{% set numpy = "2.2.1" %}
```

Comment:
Not sure if this mattered for previous builds or not, I suspect not since `numpy` is set centrally in conda-forge pinnings - but this change is more correct either way.

Reply:
It matters because we do the `<2.{N+3}` handling here, and this doesn't work with the global pinning (which now does major-only).

Reply:
The 1.14.1 package has this metadata:
```
 - numpy <2.4
 - numpy >=1.21,<3
 - numpy >=1.23.5
```

So it's already correct for previous builds (yay) - apparently `numpy` was defined in the conda-forge builds, or it would have said `<2.3`.

Anyway, this change is still helpful.